docs/website/docs/plus/getting-started/tutorial.md

```diff
@@ -0,0 +1,385 @@
+```sh
+# add a new arrow source called "sql_db_1"
+dlt source sql_db_1 add sql_database
+```
+
+This will add the new source to your `dlt.yml` file:
+
+```yaml
+sources:
+  arrow:
+    type: sources.arrow.source
+
+  sql_db_1:
+    type: sql_database
+```
+
+The corresponding credential placeholders will be added to `.dlt/secrets.toml`, but you can also define them in `dlt.yml`.
+```toml
+[sources.sql_db_1]
+table_names = ["family", "clan"]
+
+[sources.sql_db_1.credentials]
+drivername = "mysql+pymysql"
+database = "Rfam"
+username = "rfamro"
+host = "mysql-rfam-public.ebi.ac.uk"
+port = 4497
+```
+
+## Configuration and profiles
+
+dlt+ introduces a new core concept - [Profiles](../core-concepts/profiles.md), which provides a way to manage different configurations for different environments. Let's have a look at our example project. The profiles section currently looks like this:
+
+```yaml
+profiles:
+  dev: {}
+```
+
+Which means the `dev` profile is empty and by default, all the settings are inherited from the project configuration. We can inspect the current state of the project configuration by running
+
+```sh
+dlt project --profile dev config show
+```
+
+This will show the current state of the project configuration with the `dev` profile loaded. If you don't specify the `--profile` option, the `dev` profile is used by default.
+
+### Adding a new profile
+
+We can now create a new profile called `prod` that changes the location of the duckdb file we are loading to, as well as the log level of the project and the amount of rows we are loading. Please run:
+
+```sh
+dlt profile prod add
+```
+
+And change the prod profile to the following:
+
+```yaml
+  prod:
+    sources:
+      my_arrow_source:
+        row_count: 200
+    runtime:
+      log_level: INFO
+    destinations:
+      my_duckdb_destination:
+        credentials: ${tmp_dir}my_data_prod.duckdb
+```
+
+We can now inspect the prod profile. You will see that the new settings are merged with the project configuration and the `dev` profile settings.
+
+```sh
+dlt project --profile prod config show
+```
+
+### Run a pipeline with the new profile and inspect the results
+
+Now, let's run the pipeline with the `prod` profile.
+
+```sh
+dlt pipeline --profile prod my_pipeline run
+```
+
+You can now see more output in the console due to the more verbose log level, and the number of rows loaded is now 200 instead of 100. Let's inspect our datasets for each profile (assuming you still have the duckdb database file from the previous chapter).
+
+```sh
+dlt dataset --profile dev my_duckdb_destination_dataset row-counts
+dlt dataset --profile prod my_duckdb_destination_dataset row-counts
+```
+
+You will see that the number of rows loaded is now 200 instead of 100 in the prod profile.
+
+:::tip
+Profiles can also be inherited from other profiles; you can find more information in [Profiles](../core-concepts/profiles.md).
+:::
+
+### Using config files with profiles
```

Comment:
Everything below here again feels like more a reference or "features" section and not a getting started tutorial.

Reply:
Agreed, but I didn't know where to put them because it's structured as a very nice tutorial, which I didn't want to remove. I added this info in core-concepts/profiles.
If @rahuljo want to take this part to features/project this would be nice!